scripts/agent_bridge.py

```
@@ -0,0 +1,59 @@
+
+def response_for_payload(payload, provider):
+    try:
+        request = decode_agent_request(payload)
+        response = provider.plan(request)
+        return encode_agent_response(response)
+    except (ProtocolError, ProviderError) as error:
+        safe_error = error_response(error.code, str(error))
+        return encode_agent_response(safe_error)
+
+
+def serve(transport, provider, timeout, once=False):
```

Ruff (0.16.1)

[warning] Boolean default positional argument in function definition

(FBT002)